andela/pulse-mobile-app
# [#157070146] Update AppIcon

#### What does this PR do?
Updates app icon to the new chosen one.

#### How should this be manually tested?
Run the app on either iOS or android and go to App menu for either and see the app icon.

#### What are the relevant pivotal tracker stories?
[157070146](https://www.pivotaltracker.com/story/show/157070146)

#### Screenshots (if appropriate)
<img width="448" alt="screen shot 2018-04-25 at 01 56 05" src="https://user-images.githubusercontent.com/17447937/39218274-795b6832-482c-11e8-80af-1517e493daaa.png">
<img width="427" alt="screen shot 2018-04-25 at 01 56 15" src="https://user-images.githubusercontent.com/17447937/39218259-6c387596-482c-11e8-8077-a6e01614d16a.png">

#### Check List
